src/component/contents/DraftEditorBlock.react.js

```diff
@@ -204,6 +213,7 @@ class DraftEditorBlock extends React.Component {
     return (
       <div data-offset-key={offsetKey} className={className}>
         {this._renderChildren()}
```

Comment:
do we still need to _renderChildren(), or can we do all through _renderBlockMap() ?